### ISSUE TITLE
game wont even boot at all

### ISSUE THREAD

**Issue by swifto12** [zealottormunds/sh2cammod#25]
i got silent hill 2 enhanced edition, exe is named sh2pc. i booted up the game once and it worked. then after i installed the modern camera mod when i boot it the screen is black for a few seconds before getting kicked out to desktop. this is reproducible

**Comment by MrSteveOp** [comment]
same problem to me have you found a way to fix it?

**Comment by lukitaAzzz** [comment]
The same thing happened to me, I changed the compatibility, but nothing...this worked for me:
Right click on the game executable, properties, compatibility, and select the "disable full screen optimizations" box. Try that...I hope it helps.

**Comment by Morsmalleo** [comment]
> The same thing happened to me, I changed the compatibility, but nothing...this worked for me: Right click on the game executable, properties, compatibility, and select the "disable full screen optimizations" box. Try that...I hope it helps.

Yeah I got the same problem, but disabling full screen optimisations for some reason disables the mod for me.

I'm running this with Silent Hill 2: Enhanced Edition with only Zealots Cam Mod installed from this repo.

Not errors or nothing comes up, the game just doesn't start unless you remove the mod

**Comment by Morsmalleo** [comment]
> > The same thing happened to me, I changed the compatibility, but nothing...this worked for me: Right click on the game executable, properties, compatibility, and select the "disable full screen optimizations" box. Try that...I hope it helps.
> 
> Yeah I got the same problem, but disabling full screen optimisations for some reason disables the mod for me.
> 
> I'm running this with Silent Hill 2: Enhanced Edition with only Zealots Cam Mod installed from this repo.
> 
> No errors or nothing comes up, the game just doesn't start unless you remove the mod

Never mind I got it working. 

All I had to do was:

1. Download and install a fresh prepacked copy of "Silent Hill 2: Restless Dreams" without running it after installation.

2. Download and run the setup file for "Silent Hill 2: Enhanced Edition" and point it to my install location of "Silent Hill 2: Restless Dreams" (this turns the install of "SH2:RE" into and install of "SH2:EE")

3. From there I just simply waited until everything was done then I ran the game and configured the settings to my liking.

4. From there, I downloaded the Camera Mod from this repo, dragged and dropped the contents of the mod archive into the root directory of my "Silent Hill 2: Enhanced Edition" install, ran the config executable and enabled the gamepad option and then ran the game.

5. From there I had no dramas at all except for the camera mod randomly glitching out at one point which stops the player from moving the camera left and right, only up and down worked at that point but a quick save & game reset fixes that no problem.

**Comment by sparsh774** [comment]
Hey guys I was having a problem please help me out, 
I wan trynna run this bad boy with sh2 restless dreams without enhanced edition for some reasons I got some errors indicating missing files or .exe not found some shit ??
